components/metrics/metrics_tangle.go

```diff
@@ -0,0 +1,65 @@
+package metrics
+
+import (
+	"github.com/iotaledger/hive.go/runtime/event"
+	"github.com/iotaledger/iota-core/components/metrics/collector"
+	"github.com/iotaledger/iota-core/pkg/protocol/engine/blocks"
+)
+
+const (
+	tangleNamespace = "tangle"
+
+	strongTipsCount     = "strong_tips_count"
+	weakTipsCount       = "weak_tips_count"
+	blocksTotal         = "blocks_total"
+	missingBlocksCount  = "missing_block_total"
+	acceptedBlocksCount = "accepted_blocks_count"
+)
+
+var TangleMetrics = collector.NewCollection(tangleNamespace,
+	collector.WithMetric(collector.NewMetric(strongTipsCount,
+		collector.WithType(collector.Gauge),
+		collector.WithHelp("Number of tips in the tangle"),
+		collector.WithCollectFunc(func() (metricValue float64, labelValues []string) {
+			count := len(deps.Protocol.MainEngineInstance().TipManager.StrongTips())
+
+			return float64(count), nil
+		}),
+	)),
+	collector.WithMetric(collector.NewMetric(weakTipsCount,
+		collector.WithType(collector.Gauge),
+		collector.WithHelp("Number of tips in the tangle"),
+		collector.WithCollectFunc(func() (metricValue float64, labelValues []string) {
+			count := len(deps.Protocol.MainEngineInstance().TipManager.WeakTips())
+
+			return float64(count), nil
+		}),
+	)),
+	collector.WithMetric(collector.NewMetric(missingBlocksCount,
```

Comment:
Not in the same order as the constants defined above.
Please move the missing blocks metric after the total blocks metric :trollface: